src/atlas/projection/detail/MercatorProjection.cc

```diff
@@ -96,6 +98,18 @@ void MercatorProjectionT<Rotation>::lonlat2xy(double crd[]) const {
     rotation_.unrotate(crd);
 
     // then project
+    if (eckit::types::is_approximately_equal<double>(crd[LAT], 90., 1e-3)) {
```

Comment:
To the same and faster effect, could this not be replaced with:
```c++
double constexpr lat_max = 90. - 1.e-3;
if ( crd[LAT] > lat_max ) {
```

Reply:
Will do

Reply:
I've simplified the check of polar closeness